tools/run_tests/performance/README.md

```diff
@@ -209,29 +212,166 @@ https://github.com/grpc/test-infra/tree/master/config/samples
 
 The script [loadtest_config.py](./loadtest_config.py) generates LoadTest
 configurations for tests running a set of scenarios. The configurations are
-written in multipart YAML format, either to a file or to stdout.
+written in multipart YAML format, either to a file or to stdout. Each
+configuration contains a single embedded scenario.
 
-The LoadTest configurations are generated from a template. The example
-configurations above can be used as templates.
+The LoadTest configurations are generated from a template. Any configuration can
+be used as a template, as long as it supports the languages that need to be
```

Comment:
I am not sure I understand this sentence correctly. I think you mean: any load test configuration yaml file can be used as a template, as long as it follows rules set in loadtest_config.py, such as having substitution keys as ${some_filed}. Is that right?

Reply:
I think the `languages` here confused me as I don't know it is talking about the language of the test or how the template file is written.

Reply:
A template does not need to contain any substitution keys. That is why I say that any load test contiguration can be used as a template. The important part is that it must contain the languages required by the set of scenarios we intend to run (for instance, if we are generating configurations to run go scenarios, the template must contain a go client and a go server; if we are generating configurations for cross-language scenarios that need a go client and a C++ server, the template must also contain a C++ server, and the same for all other languages).

Reply:
Updated README to reflect the above.